daemon/remote/cmd/cmuxd-remote/tmux_compat_test.go

```
@@ -175,6 +177,44 @@ func TestTmuxCompatStoreRoundTrip(t *testing.T) {
 }
 
+func TestTmuxCompatStoreConcurrentMutations(t *testing.T) {
+	home := t.TempDir()
+	t.Setenv("HOME", home)
+
+	const writers = 20
+	errors := make(chan error, writers)
+	var group sync.WaitGroup
+	group.Add(writers)
+	for i := 0; i < writers; i++ {
+		i := i
```

golangci-lint (2.12.2)

[error] The copy of the 'for' variable "i" can be deleted (Go 1.22+)

(copyloopvar)